### PR TITLE
Added the max-mul operation and benchmark by Nvidia Nsight Compute

### PR DESCRIPTION
**Pull request by ArrogantGao**

### CODE REVIEW

File: benchmark/ncu/report_CuTropical.ncu-rep
Comment (GiggleLiu) [inline benchmark/ncu/report_CuTropical.ncu-rep:1]: This file is too big. We should not merge it to main. Please move it to another branch.

File: benchmark/ncu/benchmark_GemmKerenls_Tropical.jl [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# notice: this code is used to benchmark the Tropical matmul in GemmKernels.jl, which is not yet released in the latest version and only supported in julia@v1.7
+# to run the code, you need to manually download the latest version repo of GemmKernels.jl and activate the enviroment
+
+import CUDA
+import InteractiveUtils
```
Comment (GiggleLiu) [inline benchmark/ncu/benchmark_GemmKerenls_Tropical.jl:5]: What is the purpose to import InterativeUtils?

**Review by GiggleLiu — CHANGES_REQUESTED** [review]

**Comment by GiggleLiu** [comment]
A minor suggestion: `ncu` is a bad branch name, which is hard to find later. A better convention is to use something like `xzg/max-mul`.

**Comment by GiggleLiu** [comment]
The rest changes look good. All the tests pass on my machine. Good job!

**Comment by ArrogantGao** [comment]
The benchmark file is removed.
The unnecessary package ```InteractiveUtils``` is removed, I copied that dependency from ```runtest.jl``` of ```GemmKernels.jl```.
I renamed the branch as ```max-mul```, but it seems that this directly closed this PR. I will create a new PR to fix that.

**Pull request closed without merging**